src/content/1.7/development/internationalization/add-a-language.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Add a language
+aliases:
+  - /1.7/development/add_a_language
+  - /1.7/development/add-a-language
+---
+
+# Add a language
+
+Before adding a language in Prestashop, you must check if the language is available in [Crowdin](https://crowdin.com/project/prestashop-official).
+After this prerequisite, adding a language in PrestaShop has three steps :
+
+* Update JSON files in Prestashop
+* ...
+* Update i18n.prestashop.com
+  
+## Update JSON files in Prestashop
+In PrestaShop, there are two files involved in process of languages, there are :
+
+* app/Resources/all_languages.json
+* app/Resources/legacy-to-standard-locales.json
+  
+The first one is used for listing languages supported by PrestaShop in back-office.
+The second one is used for migrating locales from old versions to 1.7.
+
+### app/Resources/all_languages.json
+In this file, you need to add a new item in the JSON.<br />
```

Comment:
```suggestion
In this file, you need to add a new item in the JSON.
```

Not necessary. If you want to skip an additional line, just put a blank line 🤔 I think it works ?

Reply:
:ok_hand: